scipy/stats/tests/common_tests.py

```diff
@@ -292,6 +292,19 @@ def check_pickling(distfn, args):
     distfn.random_state = rndm
 
 
+def check_freezing(distfn, args):
+    # regression test for gh-11089: freezing a distribution fails
+    # if loc and/or scale are specified
+    if isinstance(distfn, stats.rv_continuous):
```

Comment:
does this only occur on `rv_continuous`? I saw this on my box: 
```
>>> bernoulli(p=0.5, loc=2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/rlucas/scipy-dev/scipy/scipy/stats/_distn_infrastructure.py", line 771, in __call__
    return self.freeze(*args, **kwds)
  File "/Users/rlucas/scipy-dev/scipy/scipy/stats/_distn_infrastructure.py", line 768, in freeze
    return rv_frozen(self, *args, **kwds)
  File "/Users/rlucas/scipy-dev/scipy/scipy/stats/_distn_infrastructure.py", line 432, in __init__
    self.a, self.b = self.dist._get_support(*args, **kwds)
TypeError: _get_support() got an unexpected keyword argument 'loc'
>>> 
```
but applying this same patch has the same code working with no error.

Reply:
Yeah. It's just that `scale` is not a valid keyword for `rv_discrete`.